test/tests/integration/backend.js

```diff
@@ -0,0 +1,70 @@
+import chai from "chai";
+const request = require("supertest");
+
+const { app, updateLumensCache } = require("../../../backend/app.js");
+describe("integration", function() {
+  // update caches
+  before(async function() {
+    await updateLumensCache();
+  });
+
+  describe("backend api endpoints", function() {
+    it("should return data of correct type", async function() {
+      let testCases = [
+        {
+          url: "/api/lumens",
+          wantResultType: "object",
+        },
+        {
+          url: "/api/v2/lumens",
+          wantResultType: "object",
+        },
+        {
+          url: "/api/v2/lumens/total-supply",
+          wantResultType: "number",
+        },
+        {
+          url: "/api/v2/lumens/circulating-supply",
+          wantResultType: "number",
+        },
+        { url: "/api/v3/lumens", wantResultType: "object" },
+        { url: "/api/v3/lumens/all", wantResultType: "object" },
+        {
+          url: "/api/v3/lumens/total-supply",
+          wantResultType: "string",
+        },
+        {
+          url: "/api/v3/lumens/circulating-supply",
+          wantResultType: "string",
+        },
+      ];
+
+      for (var tc of testCases) {
+        console.log("testing: ", tc.url);
+        let { body } = await request(app)
+          .get(tc.url)
+          .expect(200);
+
+        chai.expect(body).to.be.an(tc.wantResultType);
+        chai.expect(containsData(body)).to.be.false;
```

Comment:
❔ Why do we want all the endpoints to contain no data?

Reply:
whoops I switched it from `isEmpty` to `containsData` to get rid of the double negative but looks like forgot to switch the equalities as well. Fixed
https://github.com/stellar/dashboard/pull/193/commits/26a04a2ce3029ab350b462597d3ee48c6765089b